patches/0003-Implement-crypto-internal-backend.patch

```diff
@@ -1676,27 +1676,32 @@ index 00000000000000..ef5af5d956163e
 +}
 diff --git a/src/crypto/internal/backend/fips140/fips140.go b/src/crypto/internal/backend/fips140/fips140.go
 new file mode 100644
-index 00000000000000..f54d39970319af
+index 00000000000000..72f7a1644deedd
 --- /dev/null
 +++ b/src/crypto/internal/backend/fips140/fips140.go
-@@ -0,0 +1,55 @@
+@@ -0,0 +1,63 @@
 +// Copyright 2024 The Go Authors. All rights reserved.
 +// Use of this source code is governed by a BSD-style
 +// license that can be found in the LICENSE file.
 +
 +package fips140
 +
-+import "syscall"
++import (
++	"internal/godebug"
++	"syscall"
++)
 +
-+// Enabled reports whether FIPS 140 mode is enabled by using GOFIPS=1, GOLANG_FIPS=1,
++var fips140GODEBUG = godebug.New("#fips140")
```

Comment:
@qmuntal I am reading this code and I am not sure if this is a typo, or if it correct. Everywhere else I see calls to `godebug.New("fips140")` what is the meaning of a leading `#`? I also don't see anywhere for `GODEBUG=#fips140=on` to be set.

Reply:
`#` is consumed by `New`: https://pkg.go.dev/internal/godebug#New.

> To disable that panic for access to an undocumented setting, prefix the name with a #, as in godebug.New("#gofsystrace"). The # is a signal to New but not part of the key used in $GODEBUG.

Reply:
Related: upstream changed theirs from `#fips140` to `fips140` post-1.24, in https://github.com/golang/go/commit/10cef816aa9769345016c04032090ae7f5851f5c. (I don't think still having `#` should hurt, though.)